### ISSUE TITLE
Missing Moodle 4.5

### ISSUE THREAD

**Issue by jimsihk** [erseco/alpine-moodle#64]
The auto tag seems having issue that v4.5 only have the rc versions but not the released one.

Also even tagging with v4.5, the target Moodle branch is not updated and still MOODLE_404_STABLE. I have added a GitHub Action flow to handle this, you may want to have a look if interested: https://github.com/jimsihk/alpine-moodle/blob/dev/.github/workflows/major-upgrade.yml

**Comment by erseco** [comment]
Hello @jimsihk,

Thank you very much for bringing this issue to my attention! I've been reviewing your actions, and while they are a bit complex, I need a bit more time to analyze and incorporate them. Great work!

For now, I've updated the workflow to perform the correct SemVer comparison in a simpler way by using an additional action ([https://github.com/marketplace/actions/semver-utils](https://github.com/marketplace/actions/semver-utils)). I've manually updated the branch management in the Dockerfile for the time being, but I'd like to incorporate your improvement to detect it automatically in the future.

Thank you so much for your help and for contributing to improving this project!
